### PR TITLE
Elevation tile request throttling

### PR DESCRIPTION
**Pull request by markpet49**
### Description of the Change
Replicates the work that @emxsys did with layer throttling for elevation tiles. 
### Benefits
Increased performance.

### CODE REVIEW

**Comment by pdavidc** [comment]
@markpet49 This is straightforward and complements the layer work nicely.

One question: Given the upcoming ElevationCoverage class, what are your thoughts on the configuration property name WorldWind.coverageRetrievalQueueSize?

**Comment by markpet49** [comment]
@pdavidc coverageRetrievalQueueSize sounds good. How about if I make that change to the upcoming coverages branch and leave this one as is?

File: src/WorldWind.js [hunk 1]
```diff
@@ -917,7 +917,8 @@ define([ // PLEASE KEEP ALL THIS IN ALPHABETICAL ORDER BY MODULE NAME (not direc
         WorldWind.configuration = {
             gpuCacheSize: 250e6,
             baseUrl: (WWUtil.worldwindlibLocation()) || (WWUtil.currentUrlSansFilePart() + '/../'),
-            layerRetrievalQueueSize: 16
+            layerRetrievalQueueSize: 16,
+            elevationRetrievalQueueSize: 16
```
Comment (emxsys) [inline src/WorldWind.js]: Just missing a documentation comment on the new configuration value.

**Comment by markpet49** [comment]
@emxsys Thanks. Documentation added. 
@pdavidc I changed the variable name as suggested.

**Comment by pdavidc** [comment]
Thanks again for taking care of this independently @markpet49.

**Pull request merged**